extension/test/fixtures/make-payment-form.html

```diff
@@ -0,0 +1,277 @@
+<!DOCTYPE html>
```

Comment:
thanks for adding that sample page, it helps me understand the JSON output of the web components, nice PR 👍 Probably we will delete that page when we complete the e2e tests 😄

Reply:
Yes, I'm planning to move it to e2e tests when we have them.